js/package.json

```diff
@@ -82,5 +82,9 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "engineStrict": true,
+  "engines": {
+    "node": ">=18.0.0"
```

Comment:
Consider adding a comment in package.json explaining why Node.js 18+ is required. This helps other developers understand the rationale behind this requirement, especially since it's a breaking change. For example:
```json
"engines": {
  // Required for modern ES features and compatibility with key dependencies
  "node": ">=18.0.0"
}
```